docs/2_1_common_api.adoc

```diff
@@ -580,6 +580,28 @@ For Co-Simulation FMUs, additional functions are defined in <<transfer-of-input-
 
 The value of a variable may only be accessed with the respective `fmi3Get/Set{VariableType}` for its type.
 
+==== Advancing Time [[advancing-time]]
+
+This section highlights the differences of the concept of time (in general the independent variable) for the three different FMI types, ME, CS and SE.
+
+In Model Exchange, time is under the sole control of the importer and its integration algorithm.
+The model itself receives the current time to be used in its computation with <<fmi3SetTime>>.
+In fact, time is not necessarily advancing linearly as solvers might need to find zero-crossing points or the importer uses the Model Exchange FMU to find points of optimal control.
+
+In Co-Simulation, time advances in variable steps negotiated between the co-simulation algorithm of the importer and the FMU.
```

Comment:
"in variable steps" --> "in (possibly) variable steps" as in many cases fixed communication timesteps are used for "naive" co-simulation

Reply:
thanks, fixed.